graph/graphs/gen/gen_example_test.go

```diff
@@ -0,0 +1,232 @@
+// Copyright ©2021 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package gen_test
+
+import (
+	"fmt"
+	"log"
+
+	"gonum.org/v1/gonum/graph"
+	"gonum.org/v1/gonum/graph/encoding/dot"
+	"gonum.org/v1/gonum/graph/graphs/gen"
+	"gonum.org/v1/gonum/graph/simple"
+)
+
+// IDRange is an IDer that provides a set of IDs in [First, Last].
+type IDRange struct{ First, Last int64 }
```

Comment:
This type and `IDSet` look convenient enough to be included in the `gen` package.

Reply:
Moved into package.